fanappics/buoy
# CPDT-203 Update style of b-input and b-textarea

## Type of change
- [ ] Project Build
- [ ] Project Configuration

## Submission Checklist

- [x] Have you followed the guidelines in our [Contributing document](https://github.com/fanappics/buoy/blob/master/CONTRIBUTING.md)?
- [ ] Did you create appropriate tests?

## Description
Update input element and label style to match new style conventions.
Moved things around in main.styl so that associated element styles are near one another

Question about css color naming conventions:
The main purpose of variable names for colors is so that you don't have to go to each place that the color is used to change it when doing theming. If we name the colors with a visually descriptive name ($night-rider, $cerulean, etc) then we'd have to have different stylesheets for each theme. If, however, we give them a functionally descriptive name ($input-border, $focus, etc), we can just have different variable `.styl` docs that we pull in for different themes.

I included both in this PR so as not to totally rock the boat, but I would like input on this issue.

## Related Issues / Bugs / Feature Requests
https://jira.fanatics.com/browse/CPDT-203

## Test Deployment URL
https://rawgit.com/fanappics/buoy/ft-cpdt-203/docs/index.html#binput

## Deployment Notes

N/A

## Steps to Test or Reproduce

Go to the styleguide link above 
Observe that it matches the styles of input fields in https://app.zeplin.io/project/5a7a2d4c69fae4776dce32dd/screen/5a7a2dc78c79ac0755fb83a9
